polybrain.el

```diff
@@ -41,92 +41,88 @@ changed."
 (add-hook 'org-brain-visualize-text-hook 'polybrain-visualize-setup)
 
 (defun polybrain--get-point-min-max ()
+  "Get `org-mode' min position and `org-brain-visualize' max position."
   (save-excursion
-    (beginning-of-buffer)
-    (search-forward "--- Entry" nil t)
-    (beginning-of-line)
+    (goto-char org-brain--vis-entry-text-marker)
     (previous-line)
```

Comment:
`org-brain--vis-entry-text-marker` is a new addition to `org-brain`. It marks the beginning of the "--- Entry" line.